package.json

```diff
@@ -18,10 +18,12 @@
     "start:dev": "./scripts/start-dev.sh",
     "start:dev:remote": "npm run setup:dev:remote && DATA_SOURCE=remote npm run start:dev",
     "start:dev:mock": "DATA_SOURCE=mock npm run start:dev",
-    "test": "$(npm bin)/jest --coverage",
     "test:watch": "npm run test -- --watch",
     "lint": "$(npm bin)/eslint --ext .tsx,.js ./src/",
     "lint:fix": "npm run lint -- --fix",
+    "test:unit": "$(npm bin)/jest --coverage --rootDir=. --config=config/jest.config.js",
+    "test": "npm run test:unit",
+    "eslint": "$(npm bin)/eslint --ext .tsx,.js ./src/",
```

Comment:
Do we need both the `lint` script and the `eslint` one? we should just pick one and drop the other, imo. I like the consistency of naming the scripts after what they're for, not the tool (so `type-check`, `lint` and `test`, not `tsc`, `eslint` and `jest`).

Reply:
No we don't need both.  I agree with naming.